bls/src/types/public_key.rs

```diff
@@ -19,6 +19,8 @@ pub struct PublicKey {
 }
 
 impl PublicKey {
+    pub const TRUSTED_SERIALIZATION_SIZE: usize = 570;
```

Comment:
Is this the same as `PUBLIC_KEY_SIZE`? If so I would prefer to name this `SIZE`, similar to `CompressedPublicKey::SIZE`.

```suggestion
    /// The size of a serialized PublicKey in bytes.
    pub const SIZE: usize = 570;
```

Reply:
Added some docs. No, it's specific to the trusted serialization, and should generally not be used, unless you know that you can trust the place where you got the serialization from. The normal public key serialization has 285 bytes.